### PR TITLE
Fixed : Update `alt text decision tree` links to be translatable

### PR DESCRIPTION
**Pull request by narenin**
Fixes a bug raised here https://github.com/WordPress/gutenberg/issues/62038

What?

Updated alt text decision tree links to be translatable.

### CODE REVIEW

**Comment by michalczaplinski** [comment]
Thank you form working on this @narenin !

This seems fine to me but perhaps @joedolson would like to confirm?

**Comment by joedolson** [comment]
These links should have translator comments to provide context for the link. In the related core patch, you can see an example of the comment in core.

**Comment by narenin** [comment]
@joedolson I have implemented the suggestions, could you please check.

File: packages/block-library/src/cover/edit/inspector-controls.js [hunk 1]
```diff
@@ -211,9 +211,14 @@ export default function CoverInspectorControls( {
 								}
 								help={
 									<>
-										<ExternalLink href={
-												__('https://www.w3.org/WAI/tutorials/images/decision-tree/')
-												}>
+										<ExternalLink
+											href={
+												// translators: Localized tutorial, if one exists. Link includes list of existing translations.
+												__(
```
Comment (joedolson) [inline packages/block-library/src/cover/edit/inspector-controls.js:217]: This is the comment from an earlier version of the patch; we should name the source in the comment, e.g.

`// translators: Localized tutorial, if one exists. W3C Web Accessibility Initiative link has list of existing translations. `

**Review by joedolson — CHANGES_REQUESTED** [review]
Additional comment text change; applies to all files.

File: packages/block-library/src/cover/edit/inspector-controls.js (hunk 1 above)
Comment (narenin) [reply packages/block-library/src/cover/edit/inspector-controls.js:217]: @joedolson  Thanks for the feedback, I have implemented the suggestions, please check.

**Review by joedolson — APPROVED** [review]
Thanks, @narenin!

**Comment by t-hamano** [comment]
@narenin

It seems there is a conflict in one file, so could you resolve that conflict? Also, GitHub Actions should be stable on the latest trunk, so I would appreciate it if you could rebase your PR again.

Thank you!

**Comment by narenin** [comment]
@t-hamano Thanks for the feedback. I have resolved the conflict, please have a look.

**Review by t-hamano — APPROVED** [review]
@narenin Thanks for the update!

**Pull request merged**